scripts/check-elf-alignment.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+set -o pipefail
+
+progname="${0##*/}"
+progname="${progname%.sh}"
```

Comment:
can you give docs to the start of the script on how to use it and what its used for please?